services/queue/src/workerinfo.js

```diff
@@ -238,13 +238,9 @@ class WorkerInfo {
     let result;
 
     if (worker) {
-      try {
-        result = await worker.modify((entity) => {
-          entity.expires = new Date(expires || entity.expires);
-        });
-      } catch (err) {
-        throw err;
-      }
+      result = await worker.modify((entity) => {
+        entity.expires = new Date(expires || entity.expires);
+      });
```

Comment:
Would this throw is `.modify` fails?

Reply:
Yep, it's just a de-dented version of the original code, with the `try { .. } catch (err) { throw err; }` removed.